src/prerna/semoss/web/services/local/NameServer.java

```diff
@@ -305,14 +305,14 @@ public Response runPixelSync(@Context HttpServletRequest request) {
 		}
 		
 		// add the route if this is server deployment
-		String routeCookieName = Utility.getDIHelperProperty(Constants.MONOLITH_ROUTE);
+		String routeCookieName = WebUtility.inputSanitizer(Utility.getDIHelperProperty(Constants.MONOLITH_ROUTE));

```

Comment:
this is not needed - we need to sanitize user inputs, not constants/values we load from DI Helper